pages/index.js

```diff
@@ -244,7 +244,8 @@ const Home = (props) => {
                 <MailIcon className={styles.icon} />
                 <p>nusfintech.ops@gmail.com</p>
               </div>
-              <form onSubmit={handleSubmit(onSubmit)} autoComplete="off">
+              
```

Comment:
Can add a comment about the commented out section?

// TODO: Integrate form with third-party service